packages/core/src/scss/_partials/_colour-constants.scss

```diff
@@ -1,15 +1,26 @@
+/* Variants of brand colours created via below SASS functions
+   Primary, Secondary colour : Lighten/Darken (% increments of 10)
+   Neutral colour : Tint/Shade (% increments of 20)
+*/
+
+// Function to Tint a color
+@function tint($color, $percentage) {
+    @return mix(white, $color, $percentage);
```

Comment:
If you had just swapped the order of `white` and `$color` arguments in the mix function, you wouldn't have had to go through and rename all the colour variables for the neutrals. Although, I can appreciate why you did it.

Reply:
Don't know how that would have helped from renaming. Now that it's done this way, keeping it